### ISSUE TITLE
[BUG] BlockHelpers

### ISSUE THREAD

**Issue by kainlighty** [oraxen/oraxen#1154]
### I have checked...

- [X] I am using the latest version of Oraxen
- [X] I am using the latest version of any dependencies
- [X] I am using ProtocolLib 5.0
- [X] I have checked if any similar bug reports exist

### Description

```
[04:07:33 ERROR]: Could not pass event GenericGameEvent to Oraxen v1.165.0
java.lang.NoClassDefFoundError: Could not initialize class io.th0rgal.oraxen.utils.BlockHelpers
        at io.th0rgal.oraxen.mechanics.provided.gameplay.noteblock.NoteBlockMechanicListener$NoteBlockMechanicPhysicsListener.onNoteblockPowered(NoteBlockMechanicListener.java:112) ~[oraxen-1.165.0.jar:?]
        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor1277.execute(Unknown Source) ~[?:?]
        at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:75) ~[purpur-api-1.18.2-R0.1-SNAPSHOT.jar:?]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:76) ~[purpur-api-1.18.2-R0.1-SNAPSHOT.jar:git-Purpur-1632]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[purpur-api-1.18.2-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:671) ~[purpur-api-1.18.2-R0.1-SNAPSHOT.jar:?]
        at net.minecraft.world.level.Level.postGameEventInRadius(Level.java:1578) ~[?:?]
        at net.minecraft.server.level.ServerLevel.gameEvent(ServerLevel.java:1630) ~[?:?]
        at net.minecraft.world.entity.Entity.gameEvent(Entity.java:1486) ~[purpur-1.18.2.jar:git-Purpur-1632]
        at net.minecraft.world.entity.Entity.gameEvent(Entity.java:1494) ~[purpur-1.18.2.jar:git-Purpur-1632]
        at net.minecraft.world.entity.Entity.gameEvent(Entity.java:1498) ~[purpur-1.18.2.jar:git-Purpur-1632]
        at net.minecraft.world.entity.Entity.move(Entity.java:1122) ~[purpur-1.18.2.jar:git-Purpur-1632]
        at net.minecraft.world.entity.LivingEntity.handleRelativeFrictionAndCalculateMovement(LivingEntity.java:2898) ~[?:?]
        at net.minecraft.world.entity.LivingEntity.travel(LivingEntity.java:2853) ~[?:?]
        at net.minecraft.world.entity.monster.Strider.travelWithInput(Strider.java:316) ~[?:?]
        at net.minecraft.world.entity.ItemSteerable.travel(ItemSteerable.java:51) ~[?:?]
        at net.minecraft.world.entity.monster.Strider.travel(Strider.java:302) ~[?:?]
        at net.minecraft.world.entity.LivingEntity.aiStep(LivingEntity.java:3399) ~[?:?]
        at net.minecraft.world.entity.Mob.aiStep(Mob.java:658) ~[?:?]
        at net.minecraft.world.entity.AgeableMob.aiStep(AgeableMob.java:154) ~[?:?]
        at net.minecraft.world.entity.animal.Animal.aiStep(Animal.java:62) ~[?:?]
        at net.minecraft.world.entity.LivingEntity.tick(LivingEntity.java:3001) ~[?:?]
        at net.minecraft.world.entity.Mob.tick(Mob.java:420) ~[?:?]
        at net.minecraft.world.entity.monster.Strider.tick(Strider.java:363) ~[?:?]
        at net.minecraft.server.level.ServerLevel.tickNonPassenger(ServerLevel.java:1218) ~[?:?]
        at net.minecraft.server.level.ServerLevel.lambda$tick$5(ServerLevel.java:697) ~[?:?]
        at net.minecraft.world.level.entity.EntityTickList.forEach(EntityTickList.java:42) ~[purpur-1.18.2.jar:git-Purpur-1632]
        at net.minecraft.server.level.ServerLevel.tick(ServerLevel.java:675) ~[?:?]
        at net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1621) ~[purpur-1.18.2.jar:git-Purpur-1632]
        at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:522) ~[purpur-1.18.2.jar:git-Purpur-1632]
        at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1483) ~[purpur-1.18.2.jar:git-Purpur-1632]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1247) ~[purpur-1.18.2.jar:git-Purpur-1632]
        at net.minecraft.server.MinecraftServer.lambda$spin$1(MinecraftServer.java:321) ~[purpur-1.18.2.jar:git-Purpur-1632]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
```

### Steps to reproduce

You should just upgrade from 1.164.1 to 1.165.0..........

### Plugin Version

Oraxen version: 1.164.1! (I think you understand why it's old, right? THE NEW ONE DOES NOT START)

### Server Software & Version

Current: git-Purpur-1632 (MC: 1.18.2)*

### How breaking is the bug?

Breaking Bug - Plugin unusable

**Comment by Boy0000** [comment]
Thanks, this should be fixed in latest push to #1153 or posted devbuild on discord 👍

**Comment by kainlighty** [comment]
I have a paid version of the plugin and naturally I'd rather wait for the release. I believe the fix will be in the next update.

**Comment by Boy0000** [comment]
It will be in the next update yes. Just letting you know that there is a devbuild with a few fixes ontop of 1.165.0 in the discord if you want that right now 👍
